dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-cache/pom.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.apache.dolphinscheduler</groupId>
+        <artifactId>dolphinscheduler-datasource-plugin</artifactId>
+        <version>dev-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>dolphinscheduler-datasource-cache</artifactId>
+    <packaging>jar</packaging>
+    <name>${project.artifactId}</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-spi</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-datasource-api</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-common</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
```

Comment:
Did you forget to add the cache jdbc dependency?

Reply:
No, I didn't forget. I intentionally didn't add the JDBC dependency because it wasn't available in the Maven repository.

Reply:
It must be available in the maven repository and comply with apache protocol or approaching license since DS is an apache software foundation open source project.

Reply:
Sorry. Could you check if this one is okay?   [https://mvnrepository.com/artifact/com.intersystems/cache-jdbc/2.0.0](url)

Reply:
It seem like this artifact haven't been updated more than 4 years. And it does not indicate compliance with any open source protocols.

Reply:
It seems that this cache database isn't really up to the mark. I don't want all my efforts to go to waste. So, I'll modify the code to make it compatible with the new version of the iris database. Is this okay? [https://mvnrepository.com/artifact/com.intersystems/intersystems-jdbc](url)

Reply:
> Cache is the predecessor of IRIS. However, we do indeed use this old version of the Cache database.😒

This doesn't comply with apache regulations.

Reply:
I'm going to give up the merge.